src/Firebase.Auth.Tests/IntegrationTests.cs

```diff
@@ -1,17 +1,15 @@
 ﻿namespace Firebase.Auth.Tests
 {
     using System;
-
+    using System.Linq;
     using FluentAssertions;
-
     using Microsoft.VisualStudio.TestTools.UnitTesting;
-    using System.Linq;
 
     public class IntegrationTests
     {
-        private const string ApiKey = "<YOUR API KEY>";
+        private const string ApiKey = "AIzaSyBoj43kkB-PO4N8buwiIt5DKgC0vgUQlR0";// "<YOUR API KEY>";
```

Comment:
@EmilAlipiev you should remove this promptly so you do not expose your API key. You will need to make a cleaner PR before submitting.

Reply:
ops this went to wrong direction. i wanted to compare latest changes with my branch, not to send a PR. you can ignore it.